### PR TITLE
A4A > Migrations: Add steps for WordPress.com & Pressable self migration tool.

### PR DESCRIPTION
**Pull request by yashwin**
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1488

## Proposed Changes

This PR:

- Adds the self migration steps for WordPress.com.
- Adds the self migration steps for Pressable.
- Adds contact form support for newly added pages
- Adds event tracking for the buttons
- Fixes a typo
- Makes an update to the tool to fold the card when a step is marked as done or already completed. 

## Why are these changes being made?

* To add steps for WordPress.com & Pressable self migration tool.

## Testing Instructions

1) Open the A4A live link.
2) Go to the /migrations/overview page > Click the `Migrate your site` dropdown button
3) Select the `Self migrate to WordPress.com` option > Verify that the steps are matching as per the design(link in the issue). Also, verify that the cards fold when they are marked as done and opens when reset. Clicking on the contact us button on the page heading should open the contact us modal with the appropriate product selected. 
4) Repeat step 2 > Verify the same for `Self migrate to Pressabel` option.

screencapture-agencies-localhost-3000-migrations-overview-migrate-to-wpcom-2024-11-19-11_52_32

screencapture-agencies-localhost-3000-migrations-overview-migrate-to-pressable-2024-11-19-11_54_16

@madebynoam: Please check if the mobile view ok. 

Screenshot 2024-11-19 at 12 01 12 PM

Mixed state:

Screenshot 2024-11-19 at 11 51 54 AM

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/sections/migrations/primary/migrations-commissions/empty-state.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ export default function MigrationsCommissionsEmptyState( {
 									}
 								) }
 							</li>
-							<li>{ translate( 'Tag the connected sites using the button below..' ) }</li>
+							<li>{ translate( 'Tag the connected sites using the button below.' ) }</li>
```
Comment (yashwin) [inline client/a8c-for-agencies/sections/migrations/primary/migrations-commissions/empty-state.tsx:71]: Removed the extra `.`

**Review by cleacos — COMMENTED** [review]
It works well but when I click on `Go to Marketplace` it reloads the full app.

**Review by cleacos — APPROVED** [review]
About my previous comment about the `Go to Markertplace` button, the same happens with the `Tag your site` and `Add my bank information`.

Other than that, it will LGTM!

**Comment by yashwin** [comment]
> About my previous comment about the `Go to Markertplace` button, the same happens with the `Tag your site` and `Add my bank information`.
> 
> Other than that, it will LGTM!

Thanks for the review, @cleacos!c

Good catch.I missed committing some changes. It is fixed now: https://github.com/Automattic/wp-calypso/pull/96525/commits/d47db9841b3c74becda4dbdbe56f9e394a8c0740

Could you please take another look?

**Review by cleacos — APPROVED** [review]
Re-approved!

**Comment by madebynoam** [comment]
Would also be a bit nicer to set the top padding on that one to 0:

image

**Comment by madebynoam** [comment]
For mobile, I'd recommend we stick to consistent button layout, stacked looks good to me:
Also IF we could bump the title down one size on mobile that would help.

image

**Review by madebynoam — APPROVED** [review]
Other than those visual issues, this looks good to me, @yashwin!

**Comment by yashwin** [comment]
Thanks for the review, @madebynoam!

I have made all the changes you have suggested!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/17009383

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @yashwin for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/17009383).

**Pull request merged**